src/_includes/layouts/default.html

```diff
@@ -5,7 +5,7 @@
         <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
         <meta content="width=device-width, initial-scale=1.0" name="viewport">
         <link type="image/x-icon" href="{{ '/' | url }}assets/images/favicon.ico" rel="shortcut icon"/>
-        <script type="text/javascript" src="{{ '/' | url }}lib/infusion/infusion-all.js"></script>
+        <script type="text/javascript" src="{{ '/' | url }}lib/infusion/infusion-all.min.js"></script>
```

Comment:
Do you need all of Infusion? If we are only using UIO you could save some bytes by using infusion-uio.min.js instead.

Reply:
That's a good question, and probably the answer is that yes, we could just use UIO.